services/src/ogc/wcs/request.rs

```diff
@@ -77,6 +77,11 @@ pub struct GetCoverage {
     #[serde(default)]
     #[serde(deserialize_with = "from_str_option")]
     resy: Option<f64>,
+
+    // Geo Engine specific
+    #[serde(default)]
+    #[serde(deserialize_with = "from_str_option")]
+    pub nodatavalue: Option<f64>,
```

Comment:
We need to document this in our docs

Reply:
yes. we need to change a lot of docs for this PR